odoo/addons/base/tests/config/14.0.conf

```diff
@@ -34,7 +34,6 @@ log_db_level = warning
 log_handler = :INFO
 log_level = info
 logfile = 
-longpolling_port = 8072
```

Comment:
The test is about making sure that the default configuration of the oldest supported version is still compatible with master. I suggest we remove this `14.0.conf` file, checkout 16.0 and do `--config odoo/addons/base/tests/config/16.0.conf --save --stop`. The diff should ultimately be
```diff
---odoo/addons/base/tests/config/14.0.conf
+++odoo/addons/base/tests/config/16.0.conf
-longpolling_port = 8072
+gevent_port = 8072
```